app/db/seeder.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'bcrypt'
+require 'faker'
+require 'sequel'
+require 'literate_randomizer'
+
+DB = Sequel.connect('sqlite://db/development.db')
```

Comment:
Looks like only the development db is seeded and not the test db, as indicated in the Rakefile

Reply:
Yes, very good point. We will have to look how we want to approach testing, but yes I agree. Seeding the test database will also be necessary.